### DISCUSSION TITLE
Help in installation

### DISCUSSION THREAD

**Discussion by mariamendez01** [emqx/emqx#6836] [Q&A] [answered]
Hi, I need help to know what OS to use in the EMQX installation.
I am using an AWS EC2 instance which is ubuntu, but my PC OS is Windows.

What version of OS EMQX should I download and install?

**Comment by thalesmg** [comment, marked as answer]
Hi @mariamendez01 !

In general, you should download the EMQ X package that matches the OS on which you'll run it.  In your case, since the AWS EC2 instance where EMQ X will run has Ubuntu, then one of the Ubuntu packages should be downloaded for it.
